/bin/bash: /data/users2/bthapaliya/anaconda-main/anaconda3/lib/libtinfo.so.6: no version information available (required by /bin/bash)
arctrdgn002
Traceback (most recent call last):
  File "/data/users2/bthapaliya/DistributedFLExperiments/DistributedFL/fedml_experiments/standalone/subavg/main_subavg.py", line 13, in <module>
    from fedml_api.model.cv.vgg import vgg11, vgg16
ModuleNotFoundError: No module named 'fedml_api'
